packages/devtools_app/lib/src/screens/logging/logging_screen.dart

```diff
@@ -181,3 +198,34 @@ class _LoggingScreenState extends State<LoggingScreenBody>
     );
   }
 }
+
+class LoggingSettingsDialog extends StatelessWidget {
+  const LoggingSettingsDialog({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final dialogHeight = scaleByFontFactor(400.0);
```

Comment:
do we need the dialog height? if we don't have it, will this dialog just shrink to the size of its content?